gdx-jnigen-loader/src/main/java/com/badlogic/gdx/jnigen/loader/SharedLibraryLoader.java

```diff
@@ -40,80 +41,6 @@
  * @author Nathan Sweet */
 public class SharedLibraryLoader {
 
-	static public Os os;
```

Comment:
Do we maybe want to retain these fields, mark them as deprecated and just do:
`static public Os os = HostDetection.os;`

Reply:
Yeah, would make adoption a bit easier